api/prisma/schema.prisma

```diff
@@ -51,6 +51,8 @@ model User {
     Repo         Repo[]         @relation("OWNER")
     sharedRepos  Repo[]         @relation("COLLABORATOR")
     UserRepoData UserRepoData[]
+
+    codeiumAPIKey String? @default("")
```

Comment:
I think we'd better not store user tokens in the database, for security and privacy policy purposes.

Let's use the browser cache (`localStorage`) instead? And let's make all the Codeium API requests directly from the user's browser.

I'm having the same concern in #216, and I plan to use `localStorage` for GitHub token as well.

Reply:
I agree. [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) seems available across all browsers. Should we use it to store users' API keys or their tokens?

Reply:
Another strange question: if one stored his/her own token before, after he/she logs out and signs in with another account, should the stored token remains?

Reply:
In this sense, `localStorage` is not a good option to store either settings or user tokens. We should indeed use the database.